translator/source_code/cotrip_translator.py

```diff
@@ -21,7 +41,7 @@ def main():
             raise ValueError(
                 'Invalid file type. Please specify a valid Json file!') from None
         wzdx_obj = wzdx_creator(cotrip_obj)
-        location_schema = 'translator/sample files/validation_schema/wzdx_v3.0_feed.json'
+        location_schema = 'translator/sample files/validation_schema/wzdx_v3.1_feed.json'
```

Comment:
By swapping this over to the 3.1 feed are we saying we will no longer support translating messages to the 3.0 spec and are only supporting 3.1 going forward? And is that what we want?

Reply:
Briefly spoke with Tony about this in the standup, looks like at this point we are fine just producing 3.1 messages. Closing this comment.